src/js/classes/Graph.js

```diff
@@ -19,79 +21,42 @@ class CountyPoint {
   }
 
   getPositions() {
-    const xs = [], ys = [];
-    this.data.forEach(data => {
+    const xs = [],
+      ys = [];
+    this.data.forEach((data) => {
       const xDiff = this.xAxis.max - this.xAxis.min;
       const yDiff = this.yAxis.max - this.yAxis.min;
-      xs.push(`${(data.x - this.xAxis.min) / xDiff * 100}%`);
+      xs.push(`${((data.x - this.xAxis.min) / xDiff) * 100}%`);
       // svgs start Y from the top, so subtract the percentage from 100
-      ys.push(`${100 - ((data.y - this.yAxis.min) / yDiff * 100)}%`);
+      ys.push(`${100 - ((data.y - this.yAxis.min) / yDiff) * 100}%`);
     });
     return [xs, ys];
   }
 
-  renderCountyName() {
-    if (!this.showName) return;
-
-    const className = `scatter-text${this.outlier ? " outlier" : ""}`;
-    const text = document.createElementNS(SVG_NS, "text");
-    text.setAttributeNS(null, "class", className);
-    text.setAttributeNS(null, "x", this.xs[0]);
-    text.setAttributeNS(null, "y", this.ys[0]);
-    text.setAttributeNS(null, "dx", 11);
-    text.setAttributeNS(null, "dy", 3);
-    text.appendChild(document.createTextNode(this.county));
-    this.plot.appendChild(text);
-
-    this.elements.push(text);
-    text.addEventListener("mouseenter", () => this.onMouseEnter());
-    text.addEventListener("mouseleave", () => this.onMouseLeave());
-  }
-
-  renderPoints() {
```

Comment:
Oof okay you made some really big changes here that I'm going to ask you to revert. The points' render function should be in their own class; it makes little sense to render them outside of it. Can you give me reasoning for why you made these changes?

Reply:
So the reason that I pulled out those functions from the CountyPoint class is because points with different radii need to be rendered at different depths. A point with a larger radius needs to be rendered below a point with a smaller radius in order for hover stuff to work. So, the issue is that the CountyPoint class has a reference to two different points, so sorting CountyPoints by radius doesn't actually work. I essentially needed to first ungroup the CountyPoint circles, second sort those points by radius, and third render them in order.

If you want me to move renderCountyNames and renderLines back into the CountyPoint class, happy to. But when it comes to rendering the points, unfortunately the CountyPoint class can't handle that itself. Plus, I think the scatterplot class handling the rendering of it's points does actually make some sense.

Let me know what you think.